kubernetes/README.md

```diff
@@ -17,9 +17,9 @@ Python 2.7 and 3.4+
 If the python package is hosted on Github, you can install directly from Github
 
 ```sh
-pip install git+https://github.com/kubernetes-kubernetes.client/python.git
```

Comment:
so it was not working before? what caused this change?

Reply:
this is an old issue. Filed https://github.com/kubernetes-client/python/issues/952 to track the fix.